WebJobs.Script.proj

```diff
@@ -107,7 +108,11 @@
              BuildInParallel="$(BuildInParallel)"/>
   </Target>
 
-  <Target Name="PackageScriptHost" DependsOnTargets="Build">
+  <Target Name="DownloadRuntimeNugetClient">
+    <DownloadNuGet OutputFileName="$(FunctionsSiteExtensionPath)\bin\tools\nuget.exe" MinimumVersion="3.3.0" DownloadUrl="https://facavalcspn.blob.core.windows.net/public/nuget.exe"  />
```

Comment:
@brettsam I've created a new storage account, put the file up there and updated CI and the build file to use that instead.